streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamThread.java

```diff
@@ -1179,7 +1197,7 @@ private void completeShutdown(final boolean cleanRun) {
         // intentionally do not check the returned flag
         setState(State.PENDING_SHUTDOWN);
 
-        log.info("Shutting down");
+        log.info("Shutting down with cleanRun {}", cleanRun);
```

Comment:
Not sure if we need this? If the shutdown is not clean, we logged an ERROR before in `StreamThread#run()`

Reply:
I would not use variable names in log messages.

Reply:
I could avoid using variable name in the log, but still feel needed to log the flag here, as we can't control future changes to set cleanRun flag.